### PR TITLE
Remove unused double-quote between anchor symbol

### PR DESCRIPTION
**Pull request by zypeh**
As the title stated.

### CODE REVIEW

**Comment by twang2218** [comment]
I hit the same issue after upgrade to latest version, and this PR looks good to me.

File: lib/filter.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ exports.heading = function (data) {
         }, options.anchor);
 
       //  Put the anchor after the title by default, unless says otherwise
-      var link = '<a href="' + id + '" class="' + anchorOpts.style + '">"' + anchorOpts.symbol + '"</a>';
+      var link = '<a href="' + id + '" class="' + anchorOpts.style + '">' + anchorOpts.symbol + '</a>';
```
Comment (twang2218) [inline lib/filter.js]: Could you also fix the `id` problem, it's missing `#` tag, so `'<a href="' + id` should be `'<a href="#' + id`. Here is my original PR for adding the anchor: 0e09996cddec975c110d23030870b037c632e948:

```js
          if (anchorOpts.position === 'before') {
            $title.prepend(`<a href='#${id}' class='${anchorOpts.style}'>${anchorOpts.symbol}</a> `);
          } else {
            $title.append(` <a href='#${id}' class='${anchorOpts.style}'>${anchorOpts.symbol}</a>`);
          }
```

**Comment by twang2218** [comment]
@zypeh I commented the code about the missing `#` tag for the `href` link problem.

File: lib/filter.js (hunk 1 above)
Comment (zypeh) [reply lib/filter.js]: Sure! Let me get back to you later.

**Comment by zypeh** [comment]
@twang2218 Happy ~

**Pull request merged**